ocaml/idl/datamodel.ml

```diff
@@ -4771,7 +4771,7 @@ module VGPU = struct
         uid _vgpu ~lifecycle:[Published, rel_boston, ""];
         field ~qualifier:DynamicRO ~ty:(Ref _vm) ~lifecycle:[Published, rel_boston, ""] "VM" "VM that owns the vGPU";
         field ~qualifier:DynamicRO ~ty:(Ref _gpu_group) ~lifecycle:[Published, rel_boston, ""] "GPU_group" "GPU group used by the vGPU";
-        field ~qualifier:DynamicRO ~ty:String ~lifecycle:[Published, rel_boston, ""] ~default_value:(Some (VString "0")) "device" "Order in which the devices are plugged into the VM";
+        field ~qualifier:DynamicRO ~ty:String ~lifecycle:[Published, rel_boston, ""] ~default_value:(Some (VString "0")) "device" "Since Plymouth a value of 0 means auto-assign to first empty slot, valid slot is in range of [11,31] for multi-VGPU purpose";
```

Comment:
The part about 'since Plymouth' can be put into a `lifecycle` parameter like this (see example in datamodel_cluster.ml):
```
~lifecycle:[  Published, rel_boston, "Only 1 vGPU with device id 0 is supported"
               ;  Changed, rel_plymouth, "Multiple vGPUs are supported, device refers to guest PCI slot" ]
"Guest PCI slot (a value of 0 means auto-assign to first empty slot, valid slot is in range of [11,31] for multi-VGPU purpose)"
```

Reply:
thanks @edwintorok ,updated